packages/service/src/domain/commands/processors/queue-command.ts

```diff
@@ -21,11 +29,14 @@ const queueCommandDefinition: CommandDefinition = {
   key: 'queue',
   shortKey: 'q',
   processor: queueCommandProcessor,
-  helpMessage: 'Dodaje do kolejki utwór z bazy, a jeżeli go tam nie ma trakuje frazę jako YouTube ID',
+  helpMessage:
+  'Dodaje do kolejki utwór z bazy, a jeżeli go tam nie ma trakuje frazę jako YouTube ID.\n' +
+  '-n\t(next) dodaje utwór na początek kolejki',
   helpUsages: [
-    '<video name or youtube-id>',
+    '<option> <video name or youtube-id>',
```

Comment:
```suggestion
    '[option] <video name or youtube-id>',
```

nawiasem kwadratowym oznaczałem opcjonalne argumenty